README.md

```diff
@@ -10,13 +10,10 @@ This repository contains tests for Mozilla's [Bouncer website](https://wiki.mozi
 
 ## Getting involved
 We love working with contributors to fill out the test coverage for Mozilla's
-support website, but it does require a few skills. By contributing to our test
+Bouncer, but it does require a few skills. By contributing to our test
 suite you will have an opportunity to learn and/or improve your skills with
 Python, GitHub, and virtual environments.
 
-For some resources for learning about these technologies, take a look at our
-documentation on [running Web QA automated tests][running-tests].
```

Comment:
This document is more about the Selenium tests, so I think it's likely to cause confusion here.